lsa-mis/VOD_Security_Tracking
# Lra 479 vod migrate t dx ticket api to the new apigee x platform

I followed the [Migration Checklist](https://dir-test.api.it.umich.edu/guides/migration-checklist) to update the TDXTicket API.

To use the new platform API's id, secret, and URL should be updated.
I tested it in the test environment and then updated the keys and URLs to point to the API production platform.

To test that the new API works 

- find a device that is not in your local database 

> check your list of devices here http://localhost:3000/admin/devices
> some serial numbers to use: G1XJVW449F, 46LYHS1, C02ZF95GLVDL, C02M92TDFD59

- try to create a new Legacy OS Record

Also, I found a situation where DeviceManagment's message happens to be nil - fixed it.
